analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/CalculationsShouldNotOverflow.cs

```diff
@@ -86,9 +130,10 @@ public void AssignmentOperators()
         i = 2147483600;
         i *= 100;               // FIXME Non-compliant
 
+        var j = 10;
         i = 2147483600;
-        i /= 10;
-        _ = i * 100;            // FIXME Non-compliant 
+        i /= j;
+        _ = i * 100;            // Noncompliant FIXME wrong reason, i doesn't change because compoundassignment is ignored
```

Comment:
#7223 is already in review to fix this. 
Once merged, I'd rebase feature branch. And then you can rebase this PR.